signals/sender/sender_display_contains_via.yml

```
@@ -0,0 +1,4 @@
+name: "Sender: Display name contains 'via'"
+type: "query"
+source: |
+  strings.contains(sender.display_name, "via")
```

```suggestion
  regex.contains(sender.display_name, '\bvia\b')
```
swapping to regex to avoid collisions on things like "Olivia"